ethereum_history_api/circuits/get_transaction/src/main.nr

```diff
@@ -0,0 +1,19 @@
+use dep::ethereum_history_api::transaction::{get_transaction, TransactionWithinBlock, PhantomTransactionProofLen, PhantomTransactionRlpLen};
+
+global MAX_DATA_LEN = 1000;
+fn main(
+    chain_id: pub Field,
+    block_number: pub u64,
+    tx_idx: pub Field
+) -> pub TransactionWithinBlock<MAX_DATA_LEN> {
+    let phantom_transaction_proof_len: PhantomTransactionProofLen <3724>  = PhantomTransactionProofLen {}; //= 7 (MAX_TREE_DEPTH) * 532 (MAX_TRIE_NODE_LEN)
```

Comment:
I think - we can use computed values here. Let's try

Reply:
You're right. Awesome.